### PR TITLE
Dynamic Update for File Refresh Command

### PR DESCRIPTION
**Pull request by AlvinChacko**
Dynamic Update for File Refresh handler in z/OS (Issue#12376)

### CODE REVIEW

**Comment by aguibert** [comment]
Hi @AlvinChacko thanks for the PR, before we can issue a build to test this PR you'll need to sign the CLA (see https://github.com/OpenLiberty/open-liberty/blob/master/CONTRIBUTING.md)

In the meantime, tagging @brenthdaniel for code review

File: dev/com.ibm.ws.artifact.file/src/com/ibm/ws/artifact/file/FileArtifactNotifier.java [hunk 1]
```diff
@@ -503,8 +507,8 @@ public synchronized boolean setNotificationOptions(long interval, boolean useMBe
         //return false if current settings match args.
         Long compareInterval = Long.valueOf(interval);
         if (compareInterval.equals(this.interval) && ((useMBean && FileMonitor.MONITOR_TYPE_EXTERNAL.equals(this.notificationType))
-            ||
```
Comment (AlvinChacko) [inline dev/com.ibm.ws.artifact.file/src/com/ibm/ws/artifact/file/FileArtifactNotifier.java]: Whitespace change

File: dev/com.ibm.ws.artifact.file/src/com/ibm/ws/artifact/file/FileArtifactNotifier.java [hunk 2]
```diff
@@ -291,6 +291,8 @@ private synchronized void updateFileMonitorService() {
                 return;
             }
             serviceProperties.put(Constants.SERVICE_VENDOR, "IBM");
+            // Adding the com.ibm.ws.kernel.monitor.app parameter to the monitors that gets registered to use in new mbean update command
```
Comment (brenthdaniel) [inline dev/com.ibm.ws.artifact.file/src/com/ibm/ws/artifact/file/FileArtifactNotifier.java]: I don't think FileArtifactNotifier is the best place to put these changes. The container services code is SPI, so it's possible someone may set up a container and not expect it to follow the same update cycle as an application.

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#run-libby

**Comment by AlvinChacko** [comment]
#build

**Comment by AlvinChacko** [comment]
#build

**Pull request merged**